feat: showcase branding in coordinate view [#2248]

cesardlinx opened the pull request:
I added the functionality of displaying the project logo in the background when hovering over the round marker or holding shift in order to trigger the hexagon coordinate view as you can see in the gif below. It shows up with a smooth transition as requested in the issue

![ancient_beast](https://github.com/FreezingMoon/AncientBeast/assets/25573926/ab92fb8c-d308-4e33-9646-13dcf4c11bf6)

Closes #2248

My wallet address is 0xb2717FC8dFcc42A75E06A38aa273C0b9F4Cba848

DreadKnight commented:
@cesardlinx Heya! The "top" as "em" is bad, if you make the screen smaller you'll see why. It should use `px` and be a little bit under the unit queue avatars.

cesardlinx commented:
I see what you mean, please give it a look now. This is how it is looking. I changed it to px and I moved the brand a little bit under the queue as suggested.

![ezgif com-optimize (1)](https://github.com/FreezingMoon/AncientBeast/assets/25573926/7324f0ff-e4ed-41a1-aa7e-c1dccca15b39)

Let me know what do you think

DreadKnight commented:
@cesardlinx Did more testing, some issues:

1. Left side of the UI shouldn't be pushed down.
2. Logo should also scale down as well with game.
The idea is to not have logo ever overlap hexagons.
I've been scaling game diagonally unlike your testing.

cesardlinx commented:
@DreadKnight Got it! lol I didn't noticed the left side sorry. I fixed all the mentioned issues. Give it a look, let me know what do you think

cesardlinx commented:
@DreadKnight Hey! Sorry I couldn't share the gif previously of how it looks but here it is

![ancient_beast_brand6](https://github.com/FreezingMoon/AncientBeast/assets/25573926/be088d02-077c-4ae1-bf3f-1409db12ac2f)

DreadKnight commented:
@cesardlinx Alright, looks pretty ok now. No worries, gifs are always nice, but fully optional.
Merged and [sent bounty](https://www.mintme.com/explorer/tx/0x0f5c526f42bd9a9236d03776a30ec41f49f52eae0b6277f34ad15142e2f2418b) 🪙

cesardlinx commented:
Cool! 👌
